app/models/grda_warehouse/hud/enrollment.rb

```diff
@@ -93,6 +93,12 @@ class Enrollment < Base
     # NOTE: you will want to limit this to a particular record_type
     has_one :service_history_enrollment, -> { where(record_type: :entry) }, class_name: 'GrdaWarehouse::ServiceHistoryEnrollment', foreign_key: [:data_source_id, :enrollment_group_id, :project_id], primary_key: [:data_source_id, :EnrollmentID, :ProjectID], autosave: false
 
+    # Cached chronically homeless at entry
+    has_one :ch_enrollment, class_name: 'GrdaWarehouse::ChEnrollment'
+    scope :ch_needs_update, -> do
```

Comment:
Does this also need to compare the `processed_as` fields when it does exist?

Reply:
actually, this scope isn't used, removing.